### ISSUE TITLE
Add notes toggle to slides

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#282]
```
hitting 'n' should toggle notes on slides. 
```

Original issue reported on code.google.com by `ericbide...@html5rocks.com` on 11 Nov 2010 at 10:38

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 16 Dec 2010 at 10:12
- Added labels: **Type-Feature**, **Priority-P2**, **Milestone-5**
- Removed labels: **Type-Enhancement**, **Priority-Medium**, **Milestone-4**

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 18 Apr 2011 at 4:55
- Added labels: **Milestone-6**

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@gmail.com` on 1 Jun 2011 at 11:41

**Comment by GoogleCodeExporter** [comment]
```
Fixed in Revision 12eda1db7d3d.
```

Original comment by `han...@google.com` on 2 Jun 2011 at 8:28
- Changed state: **Fixed**
